### PR TITLE
Fix colors for print to PDF

### PR DESCRIPTION
**Pull request by paulathevalley**
White text will now be black in print mode. 

screen shot 2015-09-10 at 2 53 07 pm

/cc @ryan-roemer @kenwheeler

### CODE REVIEW

**Comment by ryan-roemer** [comment]
The first slide seems to now be bleeding together...

screen shot 2015-09-11 at 10 54 06 am

**Comment by ryan-roemer** [comment]
Also, I know this doesn't attempt to solve #12 but, for @kenwheeler here is Chrome print dialog with margins set to none -- there's a slight skew that accumulates by the last page...

screen shot 2015-09-11 at 10 52 46 am

**Comment by kenwheeler** [comment]
Should it be in serif font like that for print? I converted text fitting away from SVG last night, will be pushing a spectacle update this morning. SVG has an implicit line height that gets really pronounced in print views for some reason. I switched it up to use regular text with CSS3 scaling, and my print views appear to be matching the screen now.

**Comment by paulathevalley** [comment]
No, that font should not be serif. The font stacks end with `sans-serif` or `monospace`, so it appears they’re not even getting applied...?

**Comment by ryan-roemer** [comment]
@paulathevalley @kenwheeler -- Screenshot may be from disabled google fonts (I'm offline a lot right now). Let me try branch fresh again. (And maybe find some better "google doesn't load" fonts?)

**Comment by paulathevalley** [comment]
@ryan-roemer We already do have font stacks set up. If the google font isn’t loading, then at the very least, a `sans-serif` should load. 

```
  primary: "'Poppins', 'Futura', 'Century Gothic', 'Trebuchet MS', Helvetica, sans-serif",
  secondary: "'Open Sans', 'Helvetica Neue', Helvetica, sans-serif",
  tertiary: "'Anonymous Pro', Inconsolata, monospace"
```

Seeing a serif suggests the fonts aren’t being applied.

**Comment by ryan-roemer** [comment]
@paulathevalley -- Yep. Pulled branch again with fresh build and problem remains. Do you see the same as my screenshot?

**Comment by paulathevalley** [comment]
@ryan-roemer Nope, but I did pull the latest spectacle that fixes text fitting and going to push.

screen shot 2015-09-11 at 9 50 34 am
screen shot 2015-09-11 at 9 51 05 am

**Comment by paulathevalley** [comment]
Ok. I am seeing the default font from spectacle, `Open Sans Condensed` (without a font stack) getting applied to some elements, such as `<Text>`, which is explaining the serif. I'll keep digging.

File: presentation/deck.example [hunk 1]
```diff
@@ -0,0 +1,10 @@
+return(
```
Comment (ryan-roemer) [inline presentation/deck.example]: Can we have this _not_ be a separate file?

Comment (ryan-roemer) [inline presentation/deck.example]: Yep, this looks wrong:

```
return(
  <script async="" src="//i5.walmartimages.com/dfw/4ff9c6c9-1217/k2-_ce6b2cd5-7246-488c-9811-0f03b44ffd05.v325.js"/>
  <script src="//fonts.walmart.com/fqp0lia.js" type="text/javascript"/>
  <script><![CDATA[var TBRewriteHostAltWeight = 10.000000;]]></script><script src="//a14.wal.co/capabilities.min.js"/>
  <script async="" src="//i5.walmartimages.com/dfw/4ff9c6c9-2c86/k2-_1cf91030-5b66-42d4-88af-42da1d47cf61.v314.js"/>
  <script src="//www.google.com/adsense/search/ads.js"/>
  <script src="//i5.walmartimages.com/dfw/63fd9f59-dd5c/k2-_136e85ec-4952-4931-a69f-d808d78a8d47.v1.js"/>
  <script src="//i5.walmartimages.com/dfw/63fd9f59-3c31/k2-_60b82d0b-f55d-49f9-a027-6ee71de83b4b.v1.js"/>
  <script src="//a14.wal.co/cdn-test.min.js"/>
)
```

**Comment by paulathevalley** [comment]
@ryan-roemer Sorry about that, you got to see some work in progress there. @kenwheeler noticed an issue on slide 21 where the `CodePane` lives, but that should be solved now. Could you try running this again and let us know what you see?

**Comment by ryan-roemer** [comment]
Great work @paulathevalley and @kenwheeler !!! Thanks!

**Pull request merged**